communities/aston-css/community-assessment.md

```diff
@@ -0,0 +1,159 @@
+
+### Accessibility
+
+- How accessible are the current resources in your community?
+
+Very. Although we are technically a paid society, we allow anybody to attend our events. I'm hoping to keep it this way, despite the student union wanting to enforce everybody being a member.
+
+- Are meetings held during hours where people may be working?
+
+We try to vary when we hold events, but mostly stick with a Wednesday evening (and sometimes Friday evening for socials). Most students shouldn't be working around then!
+
+- Is there childcare available?
+
+No, although this is mostly not applicable to our community. As far as I'm aware this doesn't impact any members.
+
+### Interests
+
+- Is your community made up only of computer science students? Are you OK with that?
+
+We have a majority make up of CS students, but also have members from other engineering subjects and also business students. It might be nice to get a wider range, but business students in particular regularly show little interest in learning much about tech, but are there to find a sidekick for their app idea! This is of course a generalisation, but from experience quite an accurate one at Aston.
+
+- Is your community open to other topics, or are their interests very specific?
+
+Like most groups of people, they have a wide range of interests, but the one topic which brings them together is technology. I think it makes sense for us to stick with this, as it's what people expect.
+
+### Goals
+
+- Why do people get involved in your community?
+
+To get help with the course, to learn something new, and to socialise with others on their course. Those are what I can think of off the top of my head. We particularly see a lot of involvement from those who already have a good understanding of the course content, and are seeking something to stretch themselves with.
+
+- Are they students hoping to get a degree?
+
+Yes.
+
+- Are they trying to ramp-up to get a job?
+
+We do hold company sponsored tech talks (read: employment scouting events).
+
+- Are they passionate about developing, regardless of their major?
+
+Yes! In fact, we sometimes see members from outside of the CS degree being more excited about some of the stuff we're running than those doing the degree! I think this is partly because we seem to have many people doing CS at Aston who kind of drifted into it
+- they don't really have much passion and are almost just in it for the money. Maybe we should aim to increase interest in more subjects outside of the course.
+
+- What activities and goals can you set for your community based on their existing goals?
+
+Continue hosting weekly events! I was part of a push around February to get a regular schedule of events going, and I'd like to keep it up. Also make sure revision sessions run
+- these have been really useful for many members in the past, but we unfortunately didn't run them in January due to lack of people willing to host them.
```

Comment:
I think for a lot of people, it's finding the time and knowing the format, because our more experienced members know their stuff, but they're never quite sure on how to present it to a range of abilities. We have got a tech events manager this year through so this might be a task for her.